pages/[...slug].tsx

```diff
@@ -24,6 +24,7 @@ import { getQueryParamsFor } from '@/lib/params'
 import { NavProps, FooterProps } from "@/lib/types"
 import { getBreadCrumb, getLanguageLinks } from '@/lib/helpers'
 import { useReactAndShare } from '@/hooks/useAnalytics'
+import {useTranslation} from "next-i18next";
```

Comment:
Coding style nit picking: Fix the spacing here.

Reply:
Fixed.